packages/babel-plugin/src/utils/build-compiled-component.ts

```diff
@@ -57,6 +58,49 @@ const getExpression = (
   return value;
 };
 
+/**
+ * Check if 'node' is a 'inBuiltComponent'.
+ * We assume it's a 'inBuiltComponent' if it starts with a lowercase letter (i.e. div).
+ *
+ * @param node Originating node
+ */
+const isInBuiltComponent = (node: t.JSXElement): boolean => {
+  let tagName;
+
+  // Get the tag name (i.e. td) from node <td />
+  if (
+    t.isJSXIdentifier(node.openingElement.name) ||
+    t.isJSXNamespacedName(node.openingElement.name)
+  ) {
+    tagName = node.openingElement.name.name;
+  } else {
+    tagName = node.openingElement.name.property.name;
+  }
+
+  if (typeof tagName !== 'string') return false;
+
+  // Check if the first letter is lower case
+  return tagName.charCodeAt(0) >= 97 && tagName.charCodeAt(0) <= 122;
```

Comment:
I don't think this is safe as we could have a user-defined component with a lower case name. We should find an alternative way to decide if this is a dom component.

Reply:
Yup, you are right. 👍